src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/EventType.java

```diff
@@ -166,6 +166,12 @@ public enum EventType {
     REISSUE_FURTHER_EVIDENCE("reissueFurtherEvidence", 0, false),
     REMOVE_LINK_FOR_CASE("removeLinkForCase", 0, false),
     REQUEST_INFO_INCOMPLETE("requestInfoIncompleteApplication", 0, false),
+    REQUEST_CORRECTION("correctionRequested", 0, false),
```

Comment:
The non-uniformity of these names is weird. Is it intentional? Wondering whether it would be better to have them all as xxxYyyZzzRequest

Reply:
Your right, I have renamed these and a few other id's for consistency, this was partially called for a last moment name change requested by the judges on the last ticket before deploy